_posts/2017-01-30-year-in-art.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Title TK"
+date: 2017-01-30 14:21
+author: eve
+categories: [javascript]
+---
+
+Artsy's end of year features are and annual chance to walk through highlights of the year while also exploring front-end experiments. For this year's piece, designed by Owen Dodd, we pursued an interactive timeline of singular moments in the art world.
+
+The header animation, a series of transparent, sliding boxes presented a unique challenge as it required animating elements in response both to the size of other elements in the viewport, and to the client’s scroll interactions.
+
+<!-- more -->
+
```

Comment:
Here's a rough one:

![blog post](https://cloud.githubusercontent.com/assets/498212/22310971/41e28728-e31f-11e6-98dd-efabe81615f7.gif)

Hard to put into GIF form since the background is a bunch of movies 😅 I'm going to try again on a more powerful computer.